### ISSUE TITLE
New Blood not correctly implemented

### ISSUE THREAD

**Issue by Manu404** [magefree/mage#4331]
Hi,

My opponent cast New Blood (http://gatherer.wizards.com/Pages/Card/Details.aspx?multiverseid=433260) on my Telekinetic Sliver (http://gatherer.wizards.com/Pages/Card/Details.aspx?multiverseid=111085) and choose Vampire type, he gains control of the sliver, the card type is changed, but in the rule text, occurences of sliver are not replaced by vampire.

Best regards

**Comment by jeffwadsworth** [comment]
This is true, but currently, the ability text is mostly static in xmage.  The task to implement this in xmage would be monumental.

**Comment by Zerrisx** [comment]
See related issue #2082
